Sources/NetworkProtection/Networking/NetworkProtectionClient.swift

```diff
@@ -57,10 +61,17 @@ public enum NetworkProtectionClientError: Error, NetworkProtectionErrorConvertib
 struct RegisterKeyRequestBody: Encodable {
     let publicKey: String
     let server: String?
+    let country: String?
+    let city: String?
 
-    init(publicKey: PublicKey, server: String?) {
+    init(publicKey: PublicKey, 
+         server: String? = nil,
+         country: String? = nil,
+         city: String? = nil) {
```

Comment:
I'm wondering if these should not be nil by default to _force_ the caller to make sure that they're handling the location when making a request. I could see this modelled as an enum perhaps:

```
enum Server {
    case automatic // Sets nil for each value to let the backend work out what to connect to
    case server // Used for debug mode only
    case location(country: String, city: String) // Used for geoswitching
}
```